### DISCUSSION TITLE
Delayed "saving" of data in view

### DISCUSSION THREAD

**Discussion by slide** [PrismLibrary/Prism#2379] [WPF]
I have a "form" in my view that I need to bind back to the viewmodel only when a command is executed (e.g., a SaveCommand or similar). Is there an example of something similar that I can look at that does something like that, or am I maybe thinking about the problem incorrectly?

**Comment by brianlagunas** [comment]
You may be thinking about this wrong, but it is possible to update bindings explicitly. This is not a Prism related issue though. Just search your favorite search engine for explicit update of bindings.
